### ISSUE TITLE
more helpful error message if incidence format and structure issues

### ISSUE THREAD

**Issue by BGalindo-Prieto** [mrc-ide/EpiEstim#136]
firstly observed with short length incidences

**Comment by BGalindo-Prieto** [comment]
Recently observed that the incidence object format could be the reason instead of the length. More specifically, the structure/formating of incidence$counts when incidence object comes from side sources.

**Comment by annecori** [comment]
I don't think we have a reproducible example for this (and Beatriz had changed to another research field) but I suspect that if you have an incidence time series that is shorter than the time at which EpiEstim decides it's possible to start estimating Rt then there may be an error which isn't caught at the moment. @rebeccanash  could you check if that is indeed the case and write a proper error message and a test for this? thanks
